languages/en.php

```diff
@@ -154,6 +154,7 @@
 	'access:write' => "Write access",
 	'access:admin_only' => "Administrators only",
 	'access:missing_name' => "Missing access level name",
+	'access:comments:change' => "There are comments on this content, by changing the access level you will also be changing the access of those comments.  Do you want to continue?",
```

Comment:
The problem is it needs to be generic enough to encompass restricting the access too.  This would also show up if you were making a public blog visible to logged in users for example.